### ISSUE TITLE
User requests ability to disable dec compensation

### ISSUE THREAD

**Issue by GoogleCodeExporter** [OpenPHDGuiding/phd2#435]
```
Evidently, the SITech mount controller has a built-in feature that 
automatically adjusts pulse guide commands based on declination.  

```

Original issue reported on code.google.com by `bw_ms...@earthlink.net` on 22 May 2015 at 6:50

**Comment by bwdev01** [comment]
Done as part of advanced dialog rework.

**Comment by bwdev01** [comment]
Closed, implemented in 2.5.dev02 release
